explore-analyze/query-filter/languages/esql-rest.md

```diff
@@ -299,6 +299,24 @@ If the response’s `is_running` value is `false`, the query has finished and th
 }
 ```
 
+You can interrupt a running async query and collect the results that have been computed so far by using the [async stop API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-esql-async-query-stop) with the query ID.
+
+```console
+POST /_query/async/FmNJRUZ1YWZCU3dHY1BIOUhaenVSRkEaaXFlZ3h4c1RTWFNocDdnY2FSaERnUTozNDE=/stop
+```
+The query will be stopped and the response will contain the results computed so far, in the same format as the `get` API.
+
+```console-result
+{
+  "is_running": false,
+  "took": 48,
+  "is_partial": true,
+  "columns": ...
+}
+```
+This API will also return the result of the query if it has finished before the stop request was made, within the `keep_alive` period.
+You can use the `is_partial` field to determine if the results are complete or not. If `is_partial` is `true`, the results may be incomplete.
```

Comment:
```suggestion
The `is_partial` field indicates result completeness. A value of `true` means the results are potentially incomplete.
```

Again just cutting sentence length. 

I'd also like to be more precise here if possible, like what's the difference between partial and incomplete? The more explicit the better where possible.